### PR TITLE
#BDRE-136 fixing coding standards defects in fileMonitor

### PR DESCRIPTION
**Pull request by bdremadhav**

### CODE REVIEW

File: hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/RandomValueGenerator.java [hunk 1]
```diff
@@ -25,45 +25,44 @@
...
  */
 public class RandomValueGenerator {
-	
 
-	public static Random random = new Random();
-	private static final Logger LOGGER=Logger.getLogger(RandomValueGenerator.class);
-
-	
+	public static final Random RANDOM = new Random();
+	public static final Logger LOGGER = Logger.getLogger(RandomValueGenerator.class);
```
Comment (bdre-bda) [inline hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/RandomValueGenerator.java:30]: MINOR Make the "LOGGER" logger private static final. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1312)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 3 issues:
* MINOR 3 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d35fcdf1daef976e5df83c979d1fca5aff1c274a/hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/Column.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
* MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d35fcdf1daef976e5df83c979d1fca5aff1c274a/hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/RandomValueGenerator.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 3 issues:
- MINOR 3 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
- MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d35fcdf1daef976e5df83c979d1fca5aff1c274a/hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/Column.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)
- MINOR [Replace all tab characters in this file by sequences of white-spaces.](https://github.com/WiproOpenSourcePractice/openbdre/blob/d35fcdf1daef976e5df83c979d1fca5aff1c274a/hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/RandomValueGenerator.java) [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00105)

File: hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/GeneratorMain.java [hunk 2]
```diff
@@ -36,8 +37,11 @@ public static void main(String[] args) throws Exception {
...
          String outputDir=listForParams.getProperty("outputPath");
 
-        String params[] =new String[]{processId,outputDir};
+        String[] params =new String[]{processId,outputDir};
         int res = ToolRunner.run(new Configuration(), new Driver(), params);
-        System.exit(res);
+        if(res != 0)
+            throw new BDREException("Hive Generator error");
+        else
+            return;
```
Comment (bdrekapil) [inline hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/GeneratorMain.java:45]: why only return

**Comment by bdrekapil** [comment]
you have changed in datagen and your pull request says its about fileMonitor
Also please test data gen and then create pull request.

**Pull request closed without merging**